snowplow/iglu-client-embedded/schemas/com.metabase/csvupload/jsonschema/1-0-3

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Whoops, need to delete this file before merging. 

Decided to rather change the code than update the schema yet again. A bit ick but better than having two broken schemas that never worked in our history.